code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2471,40 +2471,25 @@
 		return 1
 
 	switch(volume)
-		if(1 to 5)
+		if(0.5 to 5)
 			if(ishuman(M))
 				var/mob/living/carbon/human/H = M
 				if(H.species.name != "Diona")
-					var/datum/organ/external/affecting = H.get_organ()
-					for(var/datum/wound/W in affecting.wounds)
-						spawn(1)
-							affecting.wounds -= W
-							H.visible_message("<span class='warning'>[H]'s wounds close up in the blink of an eye!</span>")
-					if(H.getOxyLoss()>0 && prob(90))
+					if(H.getOxyLoss()>0 && prob(100))
```

Comment:
I will change this tomorrow as I am about to sleep but just to clarify with you. Do I remove every instance of just "prob(100)" or do I also remove the "&&" syntax(?) before it?

Also, I'd appreciate if you could give me a short explanation on how to change things on an active PR. Do I just commit to this PR directly?